cmd/driver/install/install.go

```diff
@@ -172,30 +174,88 @@ func (o *driverInstallOptions) RunDriverInstall(ctx context.Context, driver *con
 	}
 	o.Printer.Logger.Info("found distro", o.Printer.Logger.Args("target", d))
 
-	err = driver.Type.Cleanup(o.Printer, driver.Name)
+	var (
+		dest string
+		buf  bytes.Buffer
+	)
+
+	if !o.Printer.DisableStyling {
+		o.Printer.Spinner, _ = o.Printer.Spinner.Start("Cleaning up existing drivers")
+	}
+	err = driver.Type.Cleanup(o.Printer.WithWriter(&buf), driver.Name)
```

Comment:
Idea is pretty simple: we start the spinner and to avoid called commands to print anythng to screen, we pass to them a new printer variable that prints onto a buffer.
After we stop the spinner, we are free to print all messages from the buffer.